### PR TITLE
STP-3680: Minor edits to improve documentation

### PR DESCRIPTION
**Pull request by jld-dehn-hpe**
## Summary and Scope

Sarah did a copy/edit of SAT as part of one of her bold goals. This PR adds her suggested changes to improve the documentation.

## Issues and Related PRs

* Resolves [STP-3680](https://jira-pro.it.hpe.com:8443/browse/STP-3680)
* Change will also be needed in `release/2.6`

## Testing

Lint and spell check

## Pull Request Checklist

- [x] Version number(s) incremented, if applicable
- [x] Copyrights updated
- [x] License file intact
- [x] Target branch correct
- [x] CHANGELOG.md updated
- [x] Testing is appropriate and complete, if applicable
- [x] [HPC Product Announcement](https://cray.slack.com/archives/C026TVCSXLH) prepared, if applicable

### CODE REVIEW

File: docs/usage/change_bos_version.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
-You can also configure the BOS version to use in the SAT configuration file. Do
-this under the `api_version` setting in the `bos` section of the configuration
-file. If the system is using an existing SAT configuration file from an older
-version of SAT, the `bos` section might not exist. In that case, add the `bos`
-section with the BOS version desired in the `api_version` setting.
+Configure the BOS version to use under the `api_version` setting in the `bos`
+section of the SAT configuration file. If the system is using an existing SAT
+configuration file from an older version of SAT, the `bos` section might not
+exist. In that case, add the `bos` section with the BOS version desired in the
+`api_version` setting.
```
[the remark this answers is not in the document]
Comment (haasken-hpe) [reply docs/usage/change_bos_version.md:12]: Maybe, just adding "Optionally" would help, so it says:

> Optionally configure the BSO version to use under....

**Review by haasken-hpe — APPROVED** [review]
Looks mostly good. There are a few spots where the rewording hurts the meaning just a little.

File: docs/release_notes/sat_2.5_release_notes.md [hunk 2]
```diff
@@ -39,8 +39,8 @@ line. It provides a table summarizing information for all jobs on the system.
   For more information, see
   [Hyphens in HPC CSM Software Recipe Variables](../usage/sat_bootprep.md#hyphens-in-hpc-csm-software-recipe-variables).
 
-- In `sat bootprep` input files, you can now render the value of the `playbook`
-  property of CFS configuration layers with Jinja2 templates.
+- In `sat bootprep` input files, the value of the `playbook` property of CFS
+  configuration layers with Jinja2 templates can now be rendered.
```
[the remark this answers is not in the document]
Comment (jld-dehn-hpe) [reply docs/release_notes/sat_2.5_release_notes.md:43]: Updated in next commit.

File: docs/usage/change_bos_version.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (jld-dehn-hpe) [reply docs/usage/change_bos_version.md:12]: @haasken-hpe, what do you think of the following:

```text
Another way to change the BOS version is by configuring it under the
`api_version` setting in the `bos` section of the SAT configuration file.
If the system is using an existing SAT configuration file from an older
version of SAT, the `bos` section might not exist. In that case, add the `bos`
section with the BOS version desired in the `api_version` setting.
```

File: docs/usage/sat_bootprep.md [hunk 3]
```diff
@@ -491,7 +490,7 @@ input file support rendering as a Jinja2 template and thus support variables:
   - `name`
   - `configuration`
 
-You can use Jinja2 built-in filters in values of any of the keys listed above.
+Use Jinja2 built-in filters in values of any of the keys listed above.
```
[the remark this answers is not in the document]
Comment (jld-dehn-hpe) [reply docs/usage/sat_bootprep.md:493]: @haasken-hpe, it was actually news to me that we shouldn't use "you" if there's a better alternative. In fact, that style was encouraged at my last company. It's a general tech writing rule that active voice is preferred to passive voice. Let's just leave the original wording here, but I agree that changing it makes it less clear. I prefer this alternative to using the passive voice.

[the remark this answers is not in the document]
Comment (haasken-hpe) [reply docs/usage/sat_bootprep.md]: Works for me, thanks!

File: docs/usage/change_bos_version.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (haasken-hpe) [reply docs/usage/change_bos_version.md]: That works, thanks, Jessica!

**Review by haasken-hpe — APPROVED** [review]
Looks good!

**Pull request merged**